docs/who-uses-meson-python.rst

```diff
@@ -0,0 +1,34 @@
+.. SPDX-FileCopyrightText: 2023 The meson-python developers
+..
+.. SPDX-License-Identifier: MIT
+
+.. _who-uses-meson-python:
+
+**************************
+Who uses ``meson-python``?
+**************************
+
+Here's a curated list of projects using ``meson-python``.
+
+
+.. list-table::
+   :widths: 20 80
+
+   * - `SciPy <https://github.com/scipy/scipy>`_
+     - Probably the most complex project using Meson and ``meson-python``.
+       Staring at the build definition too long may cause headaches.
+
+   * - `scikit-image <https://github.com/scikit-image/scikit-image>`_
+     - A project of medium complexity, with a lot of Cython_ code.
+
+   * - `siphash24 <https://github.com/dnicolodi/python-siphash24>`_
+     - A very simple project. It demonstrates how Meson makes it trivial to
+       compile a `CPython extension`_ written in `Cython`_ via a simple template
+       engine and link it to a library compiled from a Meson sub-project. Also
+       an example of how to use `cibuildwheel`_ to produce Python wheels for
+       several platforms.
```

Comment:
I used this little project of mine just because I know it, I'm not sure it is a good example. Feel free to pick another one.

Reply:
There aren't *that* many projects out there right now. I originally added the main ones that seemed alright to me. Anyway, it is good to have a simpler one as an example.